Extensions/Content/Dnn.PersonaBar.Extensions/WebApps/Pages.Web/src/components/dnn-persona-bar-page-treeview/src/PersonaBarPageTreeviewInteractor.jsx

```diff
@@ -664,6 +666,31 @@ class PersonaBarPageTreeviewInteractor extends Component {
                 });
             }
         });
+
+        const waitForChildList = (timeoutTime, callback) => new Promise((resolve, reject) => {
+            const check = () => {                
+                if(this.state.isChildLoaded) {
+                    callback();
+                    resolve();
+                } else if((timeoutTime -= 100) < 0) {
```

Comment:
space between if and parenthesis is missing

Reply:
Updated as suggested, thank you.